search.py

```diff
@@ -0,0 +1,72 @@
+from rich import print
+from rich.panel import Panel
+from rich.prompt import Prompt
+from rich.text import Text
+from models.pokemon import pokemon
+from pokemon_list import pokemon_list
+
+from interaction.db_interaction import database_manager
+db = database_manager()
+
+class search(pokemon_list):
+
+    def __init__(self):
+        pass
+
+    def run_page(self, current_user):
+        title = Text("What would you like to search for today?", justify="center")
+        print(Panel(title, style='bold', title='Search', padding=(1, 1)))
+        print()
+        pokemon_name = self.get_user_pokemon()
+        admin_username = "Admin"
```

Comment:
I'm not sure if we should do it dependant on admin, especially if there hasn't been an import script ran